src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -0,0 +1,202 @@
+a separate GitHub repository within the
+`visit-dav GitHub organization. <https://github.com/visit-dav>`_
+
+The Primary's Role
+------------------
+
+The primary's role is to respond, within one hour's time, to each incident
+that occurs during that week including those that came in during the preceding
+weekend/holiday. Other developers who may be contacted directly by customers
+should redirect such inquires to the primary. The primary's goal is to *wrap-up*
+all incidents by the end of their week. If an issue cannot be wrapped-up, it
+gets handed-off to next week's primary. Handoff's are managed formally with an
+email to customer and the next week's primary and backup contact information.
+
+For each new incident, the primary should file an issue ticket in the
+`visit-live-incidents <https://github.com/visit-dav/visit-live-incidents/issues>`_
+repository and assign themselves. When the incident is resolved, the associated
+issue should be closed. Primary should endeavor to capture all relevant
+information and communications involving the incident in this issue. For users
+who do not have GitHub accounts, this will require the primary to do a modest
+amount of cutting and pasting between email and GitHub issues.
+
+The goal of the primary is to reach a *resolution* of each incident. Resolution
+often involves one or more of the following activities.
+
+  * Answering a question or referring user to documentation.
+  * Determining if user's issue is known (e.g. an issue ticket already exists).
+  * Updating a known issue with additional information from this user, perhaps
+    reprioritizing the issue or putting the issue ticket back into the
+    UN-reviewed state for further discussion at a VisIt_ project meeting.
+  * Diagnosing the incident.
+  * Identifying and subsequently fixing any *low-hanging fruit*.
+  * Developing a reproducer for developers.
+  * Developing a work-around for users.
+  * Identifying and filing a *new* coding issue in the main source repository.
+
+In particular, please take note that *resolution* does not always mean a
+customer's incident is able to be addressed favorably within the constraints of
+our customer response processes. Sometimes, the best that can be achieved is
+the filing of a highly informative issue ticket to be prioritized, scheduled
+and ultimately resolved as part of normal weekly development meetings.
+
+Another role of the primary is to use any time not working active incidents to
+fix *low-hanging fruit* issues; either those the primary is currently managing
+or those left undone from previous incidents. For example, if an incident
+reveals incorrect or missing documentation, the primary should work to fix such
+issues within his/her week. As a rule of thumb, low-hanging fruit is considered
+to be anything that primary believes is fixable within a half-day's (4 hours)
+worth of effort. 
+
+When primary has challenges any of the paths to resolution of an incident, s/he
```

Comment:
The grammar here is a little rough. Maybe a word is missing between "challenges" and "any"?